### PR TITLE
feat: Integration tests run when pr is approved and not draft

### LINKED ISSUE #1147: Integration tests for iOS
**Author the user story for this feature**

As a flank developer, I want to have ios integration tests so I can be sure everything is ok.

**Is your feature request related to a problem? Please describe.**

We turned off IT tests for iOS in this pr: #1129. Now integration tests running on every commit in a pull request this cause the quota limit exceeded.

**Describe the solution you'd like**

I think we can move integration tests to merge to master action.

### PR DESCRIPTION
**Pull request by adamfilipow92**
Fixes #1147

To avoid test lab quota exceed integration tests should run when pr is ready to review and have more than 2 approvals.

### CODE REVIEW

File: .github/workflows/integration_tests.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+name: integration-tests
+
+on:
+  pull_request_review:
+    types: submitted
+
+jobs:
+  build:
+    runs-on: [ubuntu-latest, macos-latest]
```
Comment (Sloox) [inline .github/workflows/integration_tests.yml:9]: we have decided that we can run this on windows wsl. does this take it into account?

**Review by Sloox — CHANGES_REQUESTED** [review]
Please check the windows wsl integration tests.

File: .github/workflows/integration_tests.yml (hunk 1 above)
Comment (piotradamczyk5) [reply .github/workflows/integration_tests.yml:9]: You must set up some steps for using WSL, so it will be easier to make a separate file for it.

Comment (adamfilipow92) [reply .github/workflows/integration_tests.yml:9]: No, it's running on ubuntu or mac, not on wsl. I will add my changes to wsl too. Thanks!

Comment (adamfilipow92) [reply .github/workflows/integration_tests.yml]: Done

**Review by piotradamczyk5 — APPROVED** [review]

**Review by Sloox — APPROVED** [review]

**Pull request merged**